### ISSUE TITLE
integrate sign in / session flow with nano ledger

### ISSUE THREAD

**Issue by jbibla** [cosmos/voyager#378]
waiting for firmware developement.

**Comment by jolesbi** [comment]
check it out ya'll!

https://github.com/cosmos/ledger-cosmos

**Comment by faboweb** [comment]
Blocked on missing cross compilation ability of the ledger code in the SDK.
Closing in favor of #1338
